app/components/uploadlaunchoverlay/uploadlaunchoverlay.less

```diff
@@ -39,10 +39,18 @@
 .UploadLaunchOverlay-title {
   font-size: @font-size-large;
   font-weight: 700;
-  margin-bottom: 10px;
+  margin-bottom: 15px;
 }
 
 .UploadLaunchOverlay-text {
   font-size: @font-size-small;
   font-weight: lighter;
 }
+
+.uploadDismiss {
+  font-size: 30px;
+  font-weight: 500;
+  float: right;
+  color: #6d6d6d;
+  padding-right: 5px;
```

Comment:
The hover style is somewhat incomplete:
![image](https://user-images.githubusercontent.com/3166747/40727414-d5f60dc4-63f5-11e8-9dd5-013f33dc7009.png)

Let's remove the underline on hover, and set the cursor to use a pointer;  I've also, in the example below, used absolute positioning instead of floats/padding, which is more predictable.

I think we should rename the class and move this to `modaloverlay.less`, as I could see this being useful for other modals at some point:
```
.ModalOverlay-dismiss {
  font-size: 30px;
  font-weight: 500;
  color: #6d6d6d;
  cursor: pointer;
  position: absolute;
  top: 16px;
  right: 20px;

  &:hover {
    text-decoration: none;
  }
}
```

You will also need to change the `uploadDismiss` classname in the corresponding components and tests (`uploadlaunchoverlay.js` and `uploadlaunchoverlay.test.js`).